network/gossip/libp2p/test/topology_test.go

```diff
@@ -12,31 +11,42 @@ import (
 	"github.com/stretchr/testify/require"
 	"github.com/stretchr/testify/suite"
 
+	"github.com/onflow/flow-go/crypto"
+	"github.com/onflow/flow-go/engine"
 	"github.com/onflow/flow-go/model/flow"
 	"github.com/onflow/flow-go/model/flow/filter"
 	"github.com/onflow/flow-go/network/gossip/libp2p/topology"
 	"github.com/onflow/flow-go/utils/unittest"
 )
 
-// TopologyTestSuite tests the bare minimum requirements of a randomized
-// topology that is needed for our network. It should not replace the information
-// theory assumptions behind the schemes, e.g., random oracle model of hashes
```

Comment:
Since our topology is no longer based on any specific randomized assumption. We no longer tied to a random oracle distribution. It is a guaranteed connected topology in a deterministic way.